[13.0][MIG] purchase_invoice_plan (#936)
OCA/purchase-workflow: purchase_invoice_plan/readme/USAGE.rst

- Create new purchase quotation as per normal process
- Select option "Use Invoice Plan", a new Invoice Plan tab will appear
- Click on "=> Create Invoice Plan" link to open invoice planning wizard
- Do plan for number of installment, start date and interval
- Double check that each installment has correct plan percentage

> - After confirm purchases order, we have new option to "Create Bill by Plan"

Reviewer: ```suggestion
- Confirm Order > Receive Products, see new option to "Create Bill by Plan"
```

Reviewer: this is my mistake, I config product purchase policy is `On received quantities`
it should be `On ordered quantities`
